docs/api.md

```diff
@@ -331,6 +331,66 @@ is explicitly specified and the runtime is Pyodide.
 
 The technical details of how this works are [described here](../user-guide/ffi#to_js).
 
+### `pyscript.fs`
+
+!!! danger
+
+    This API only works in Chromium based browsers.
+
+An API for mounting the user's local filesystem to a designated directory in
+the browser's virtual filesystem. Please see
+[the filesystem](../user-guide/filesystem) section of the user-guide for more
+information.
+
+#### `pyscript.fs.mount`
+
+Mount a directory on the user's local filesystem into the browser's virtual
+filesystem. If no previous transient user activation has taken place, this
+function will result in a minimalist dialog to provide the
+[transient user activation](https://developer.mozilla.org/en-US/docs/Glossary/Transient_activation).
+
+This asynchronous function takes four arguments:
+
+* `path` (required) - indicating the location on the in-browser filesystem to
+  which the user selected directory from the local filesystem will be mounted.
+* `mode` (default: `"readwrite"`) - indicates how the code may interact with
+  the mounted filesystem. May also be just `"read"` for read-only access.
+* `id` (default: `"pyscript"`) - indicate a unique name for the handler
+  associated with a directory on the user's local filesystem.
```

Comment:
in here we're lacking extra context ... such as the fact "*it also allows users to select different folders and mount these at the very same virtual path level previous folders where mounted before*"